src/main/java/org/gridsuite/modification/server/modifications/BatteryCreation.java

```diff
@@ -163,86 +136,11 @@ private void addExtensionsToBattery(BatteryCreationInfos batteryCreationInfos, B
         }
         reportBatteryConnectivity(batteryCreationInfos, subReporter);
         Reporter subReporterLimits = reportBatteryActiveLimits(batteryCreationInfos, subReporter);
-        createBatteryReactiveLimits(batteryCreationInfos, battery, subReporterLimits);
+        createReactiveLimits(batteryCreationInfos, battery, subReporterLimits);
```

Comment:
ModificationUtils.getInstance().createReactiveLimits ? and no static keyword in this method call ?